### ISSUE TITLE
[kudu1] No key input accepted

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#522]
# Problem
- After recent factory repair https://github.com/130s/hut_10sqft/issues/440, and upgrading all deb pkgs, kudu1 stopped reacting to keyboard input after OS reboot.
- Can't even log on to each user at the display manager's login console. Issue already happening there, meaning this is irrelevant to each user account's setting.
- Only exceptional key is caps key, as assigned in Ubuntu's k/b config (i.e. caps-ctrl are swapped), seems to be reacting.

# Impact
This makes the laptop headless, which I don't intend. And I'm planning kudu1 to be one of main daily computers. So this is critical.

**Comment by 130s** [comment]
I don't find any unusual behavior when typing in over `ssh`.

**Comment by 130s** [comment]
Screen saving mode got canceled when I touched a random key. So Ubuntu IS detecting key stroke.

**Comment by 130s** [comment]
<details><summary>syslog is surprisingly short for today (8:40 is likely when I turned the power on).</summary>

```
Feb 22 08:40:26 fussili-kudu1 rsyslogd:  [origin software="rsyslogd" swVersion="8.32.0" x-pid="1360" x-info="http://www.rsyslog.com"] rsyslogd was HUPed
Feb 22 08:40:28 fussili-kudu1 colord[3234]: failed to get session [pid 11843]: No data available
Feb 22 08:40:34 fussili-kudu1 anacron[9978]: Job `cron.daily' terminated
Feb 22 08:40:34 fussili-kudu1 anacron[9978]: Normal exit (1 job run)
Feb 22 08:40:42 fussili-kudu1 system76-daemon[1352]: 2021-02-22 08:40:42,460  INFO  saving brightness at 137
Feb 22 08:45:18 fussili-kudu1 boinc[2506]: No protocol specified
Feb 22 08:45:18 fussili-kudu1 boinc[2506]: dir_open: Could not open directory '/dev/input/mice' from '/var/lib/boinc-client'.
Feb 22 08:45:18 fussili-kudu1 boinc[2506]: 22-Feb-2021 08:45:18 [---] Resuming network activity
Feb 22 08:45:19 fussili-kudu1 boinc[2506]: No protocol specified
Feb 22 08:47:55 fussili-kudu1 boinc[2506]: message repeated 157 times: [ No protocol specified]
Feb 22 08:47:56 fussili-kudu1 boinc[2506]: No protocol specified
Feb 22 08:53:15 fussili-kudu1 boinc[2506]: message repeated 317 times: [ No protocol specified]
Feb 22 08:53:16 fussili-kudu1 boinc[2506]: 22-Feb-2021 08:53:16 [---] Suspending network activity - computer is in use
Feb 22 08:53:22 fussili-kudu1 system76-daemon[1352]: 2021-02-22 08:53:22,531  INFO  saving brightness at 464
Feb 22 08:53:32 fussili-kudu1 system76-daemon[1352]: 2021-02-22 08:53:32,532  INFO  saving brightness at 137
Feb 22 08:54:49 fussili-kudu1 systemd[1]: Started Session 13 of user nfussili.
```
</details>

I see `dir_open: Could not open directory '/dev/input/mice' from '/var/lib/boinc-client'.`.

**Comment by 130s** [comment]
> I see `dir_open: Could not open directory '/dev/input/mice' from '/var/lib/boinc-client'.`.

I misread `boinc` with `bionic` :p...It is about the boinc app and it seems to be resolved https://github.com/BOINC/boinc/issues/2335. That shouldn't be causing my issue?

**Comment by 130s** [comment]
<details><summary>Tried xorg-input thing but no.</summary>

https://askubuntu.com/questions/1033767/keyboard-not-working-after-update-to-18-04
```
$ sudo apt-get install xserver-xorg-input-all
:
The following NEW packages will be installed:                                                                                                                                                            
  xserver-xorg-input-all                                                                                                                                                                                
0 upgraded, 1 newly installed, 6 to remove and 0 not upgraded.
Need to get 4,112 B of archives.                              
After this operation, 383 kB disk space will be freed.

$ startx 
xauth: (stdin):2:  unknown command "fd834d67c9d4b0a628110cccddbde93b"

/usr/lib/xorg/Xorg.wrap: Only console users are allowed to run the X server

Invalid MIT-MAGIC-COOKIE-1 keyxinit: giving up
xinit: unable to connect to X server: Resource temporarily unavailable
xinit: server error
Couldn't get a file descriptor referring to the console
$ 
```
</details>

**Comment by 130s** [comment]
I found it stranger than I thought.

- IIRC, first time I observed was AFTER I was able to log on at the display manager and moved the cursor over `synergy`. 
- At the password input field that I thought was display manager's login console, I clicked "Switch Account", which lead to another view where I see the list of existing users with my main account selected. This is expected. Then, there I was able to type. But typing in the password would lead to the same previous view where only the password was requested and no k/b input was taken.

Anyways, now, after https://github.com/130s/hut_10sqft/issues/522#issuecomment-783398546, I rebooted OS and it seems to be working. Maybe the missing `xserver-xorg-input-all` was the key?
